models/models.endpoints.js

```diff
@@ -0,0 +1,8 @@
+const fs = require("fs/promises");
+const db = require("../db/connection");
+exports.readEndpoints = () => {
+  return fs.readFile("endpoints.json").then((endpoints) => {
+    const parsed = JSON.parse(endpoints);
+    return parsed;
+  });
+};
```

Comment:
This is great but it can be moved to the controller. The model is reserved for functions that access the database. 

This endpoint does not require a model.